src/Sql/Sql/help/Set-AzSqlInstance.md

```diff
@@ -290,6 +325,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -SkuName
+The SKU name for the instance e.g. 'BC_Gen4'.
```

Comment:
As Gen4 is becoming deprecated, maybe it's better to have BC_Gen5 here

Reply:
It is much better. Fixed